tasks/mssql_conf_setting.yml

```diff
@@ -0,0 +1,37 @@
+- name: Check if the {{ __mssql_conf_setting }} setting is enabled
+  command: >-
+    /opt/mssql/bin/mssql-conf get {{ __mssql_conf_setting }}
+  ignore_errors: true
```

Comment:
Under what conditions will this command issue an error?  When the setting is not set?

Reply:
I checked and `mssql-conf get` does not return errors. I tried getting a defined, undefined, and not existing setting and it never returned `1`. So I am removing ignore_errors from all `get` tasks, thank you for this catch.